### ISSUE TITLE
Add Custom Events to Logtivity Logs?

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#246]
Logtivity makes it easy to log custom events:
https://logtivity.io/docs/log-custom-events/

Some initial ideas:

- The details of our own plugin settings changes (see note below).
- Each time a snapshot is sent back (including whether or not it was successful -- right now there's no notification if an API call isn't successful)
- Every time there's a Cloudflare cache clear (manual and/or automatic?)
- When an IP is allow listed on the Sucuri Firewall

For the details of our own plugin settings changes - right now it does automatically log that the settings were changed, but no other details:

![image](https://user-images.githubusercontent.com/24479276/232335840-515c4fe1-25c8-45fa-a414-fb065667749a.png)

**Comment by CNoemi94** [comment]
Is this still needed? 

How would this affect or work with the events in the Relay Server?

**Comment by CNoemi94** [comment]
@blogtutor
Is this still needed?

How would this affect or work with the events in the Relay Server?

**Comment by blogtutor** [comment]
Yes we should revisit this and see if there's anything we want more granular logging, where logtivity would be the right tool to do so.

Logtivity can store on its own, AND can send a webhook which will end up as an Event in our Relay Timeline, if we want.
